### DISCUSSION TITLE
圣三一学院

### DISCUSSION THREAD

**Discussion by Manyoshu** [share121/inter-knot#1965] [其他]
![1692254653570](https://github.com/user-attachments/assets/3d7c8f0e-2159-41c9-9338-7e39089e81a8)
基沃托斯三大学院之一，其学生会名为“茶话会”。
设有自警团、正义实现委员会、救护骑士团等许多社团。
总体来说重视纪律和文化，学生们大多诚实善良，给人不谙世事的大小姐印象，内部的治安状况有序而稳定。
自古以来就在基沃托斯推崇“文武双全”，是一所充满历史和传统气息的学园。[1-Vol3.1.3 茶话会]
当然，圣三一并不是完美的。在风平浪静的学院氛围下，也潜藏着诸多问题……
和格赫娜在传统上是敌对关系。

圣三一综合学园最初是由许多分派系的学校组成的，包括帕特尔（圣父）、菲利乌斯（圣子）、桑库特斯（圣灵）、阿里乌斯等分派系。这些分派之间存在敌对关系，并经常爆发纷争。然而，在第一次公会议后，他们决定停止争斗，友好相处，建立了茶话会，形成了现在的圣三一综合学园。

权力机构
茶话会是圣三一综合学园的学生会，学生会长每代都由多人担任。起源于第一次公会议，以帕特尔、菲利乌斯、圣库特斯…这三所学园的代表为首，开设了“茶会”，产生了走向和解的潮流。从那以后，圣三一的学生会就被统称为“茶话会”各派系的代表轮流担任“主办者（host）”。现阶段，茶会的主办者（host）为「桐藤渚」，其余二位成员分别是「圣园未花」和「百合园圣娅」。

词源解释：圣三一
Trinity（トリニティ）即三位一体，来自拉丁语：trinus ‘threefold’，是基督教的根本教义。
爱任纽（Irenaeus，约137～202年）是三位一体论（简称“三一论”）的先驱。谓上帝本体为一，但又是圣父、圣子（耶稣基督）和圣灵（或称“圣神”）三位。即上帝只有一体，但在位格上又分成圣父、圣子和圣灵；三者虽有特定位份，却完全同属一个本体，同为一个独一真神，而不是三个神，但又非只有一位。教会认为，这是“奥秘”的“启示”，不可能靠理性来领悟，只能靠信仰来接受。源出于《马太福音》和《哥林多后书》。
三位一体，谓上帝独一，但同时具有圣父、圣子、圣灵三个位格。三者虽位格有别，而本质绝无分别，同受钦崇，同享尊荣，同为永恒。三位一体把《圣经》中所记述的对上帝本性的见证的三个中心要素放在一起，即：神创造世界、神藉着耶稣基督救助人类、神藉着圣灵与基督徒和教会同在。三位一体的教义断定这三个要素系于同一位神。它不是说有一位神创造世界，另一位施行救赎，又有第三位与基督徒和教会同在，而是说同一位神在一贯的历史过程中做这三件最重要的事情。

**Comment by lllliufeng** [comment]
谁问你了？
Who asked you?
誰があなたに聞きましたか?
Кто тебя спрашивал?
Qui t'a demandé?
Wer hat dich gefragt?
¿quién te preguntó?
Chi te l'ha chiesto?

**Comment by schalesensei843** [comment]
sensei，绳网好玩吗[狗头]
